src/features/mcda/utils/openMcdaFile.ts

```diff
@@ -10,6 +11,29 @@ const input = (() => {
   return input;
 })();
 
+function createMCDAConfigFromJSON(json: Partial<MCDAConfig>): MCDAConfig {
+  const result: Partial<MCDAConfig> = { ...json };
+  if (!result.version || result.version < 4) {
+    throw new Error(i18n.t('mcda.error_wrong_mcda_version'));
+  }
+  if (
+    !result?.layers?.length ||
+    !(
+      result?.colors?.type === 'sentiments' ||
+      result?.colors?.type === 'mapLibreExpression'
+    )
```

Comment:
Can be simplified to smth like this 
`!['sentiments', 'mapLibreExpression'].includes(result?.colors?.type)`

Reply:
![image](https://github.com/konturio/disaster-ninja-fe/assets/9570735/eb75e418-b945-490e-ac09-fa9e4fffa1d9)
This simplification doesn't exactly work without additional checks or adding empty string just for the sake of having correct type. I will leave it as is for now